src/app/technical-record/vehicle-summary/vehicle-summary-psv/vehicle-summary-psv.component.spec.ts

```diff
@@ -12,17 +12,22 @@ describe('PsvVehicleSummaryComponent', () => {
   beforeEach(async(() => {
     TestBed.configureTestingModule({
       imports: [SharedModule],
-      declarations: [ VehicleSummaryPsvComponent ],
-      schemas: [CUSTOM_ELEMENTS_SCHEMA],
-    })
-    .compileComponents();
+      declarations: [VehicleSummaryPsvComponent],
+      schemas: [CUSTOM_ELEMENTS_SCHEMA]
+    }).compileComponents();
 
     fixture = TestBed.createComponent(VehicleSummaryPsvComponent);
     component = fixture.componentInstance;
-        component.activeRecord = TESTING_UTILS.mockTechRecord({
+    component.activeRecord = TESTING_UTILS.mockTechRecord({
       approvalType: 'approval',
       variantNumber: '123',
       ntaNumber: '4566',
+      speedLimiterMrk: true,
+      tachoExemptMrk: true,
+      euroStandard: '6',
+      fuelPropulsionSystem: 'gas',
+      numberOfWheelsDriven: 4,
+      emissionsLimit: 23,
       vehicleClass: TESTING_UTILS.mockBodyType()
```

Comment:
use this instead `vehicleClass: {
        code: '2',
        description: 'motorbikes over 200cc'
      }`